tests/integration/framework/framework.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/dapr/dapr/tests/integration/framework/process"
 )
 
+var Domain = "dapr.io"
```

Comment:
I have this exported in kit as `Domain`, you can use kitErrors.Domain

Reply:
For integration tests I kind of prefer having the expected values on the test side, so we can catch if something changes in the code and prevent possible breaking changes to clients. This is especially true for error codes, but I think it applies to ErrorInfo details too.

Reply:
I agree that this doesn't belong here- at a minimum this should be a `const` and not live in `framework.go` as it is test data. Ideally, we would use `"dapr.io"` string literals in test assertions.